### PR TITLE
Remove instanciation of Style at user level

### PR DESCRIPTION
**Pull request by ftoromanoff**
Purpose:
Move the instanciation of new itowns.Style to Layer instead of the user level to prepare for a rework on Style.

Modifications:
- remove new itowns.Style() on example and tutorial files
- change JSdoc for Style.js

### CODE REVIEW

File: src/Core/Style.js [hunk 1]
```diff
@@ -410,11 +557,15 @@ class Style {
     }
 
     /**
+<<<<<<< HEAD
      * Map drawing properties style (fill, stroke and point) from context to object.
+=======
+     * Map drawing properties style (fill, stroke, point, text and icon) from context to object.
+>>>>>>> 8effd638 (doc(style): JSDoc for StyleOptions)
```
Comment (gchoqueux) [inline src/Core/Style.js]: could you resolve?

File: src/Layer/ColorLayer.js [hunk 2]
```diff
@@ -84,7 +83,7 @@ class ColorLayer extends RasterLayer {
         deprecatedColorLayerOptions(config);
         super(id, config);
         this.isColorLayer = true;
-        this.style = new Style(config.style);
+        this.style = config.style;
```
Comment (gchoqueux) [inline src/Layer/ColorLayer.js:87]: the line could be removed?

File: src/Core/Style.js (hunk 1 above)
Comment (ftoromanoff) [reply src/Core/Style.js]: done

File: src/Layer/ColorLayer.js (hunk 2 above)
Comment (ftoromanoff) [reply src/Layer/ColorLayer.js:87]: Layer doesn't have a style properties.
In this PR we added the fact that if there is a style properties in options at Layer we change it to a new Style(config.style).

**Review by gchoqueux — APPROVED** [review]

**Pull request merged**